docs/docs/admin/teamforcapella/migration.md

```diff
@@ -36,8 +41,19 @@
        on the old TeamForCapella server.
        ![Disable backup option](../../user/tools/capella/migration/backup-migration.png)
 
-1.  Export the model to the new TeamForCapella server via the Export wizzard
+1.  Export the model to the new TeamForCapella server via the Export wizard
     from TeamForCapella:
     [Export a model to TeamForCapella](../../user/tools/capella/teamforcapella/export/index.md)
 1.  Recreate the backup pipeline:
-    [Set up a TeamForCapella backup](../../user/projects/models/backups/setup.md)
+    [Set up a TeamForCapella backup](../../user/projects/models/backups/setup.md).
+    Start a pipeline run and verify that the changes are included in the Git
+    repository.
+1.  Use the [capella-diff-tools](https://github.com/DSD-DBS/capella-diff-tools)
+    and compare the commit after the migration with the commit before the
+    migration. Verify that the changes are correct.
+1.  Communicate to all users that the model has been migrated to a new version.
+    Users have to use sessions of the newer Capella version after the
+    migration. Since the connection details have changed, all users have to
+    delete the old `model.team` project from their personal workspace and
+    reconnect to the TeamForCapella repository:
+    [Connect to a TeamForCapella repository](../../user/tools/capella/teamforcapella/export/index.md)
```

Comment:
Everything (from "Since (...)" onwards) can be removed, when we have the 2nd step of my proposal in the previous comment.